src/ShiftedProximalOperators.jl

```diff
@@ -46,12 +48,22 @@ end
 
 Set the trust-region radius of a shifted proximable function to Δ.
 This method updates the indicator of the trust region that is part of ψ.
+
+    set_radius!(ψ, l, u)
+
+Set the trust-region l and u parameters if it is defined as a box [l,u].
 """
 function set_radius!(ψ::ShiftedProximableFunction, Δ::R) where {R <: Real}
   ψ.Δ = Δ
   return ψ
 end
 
+function set_radius!(ψ::ShiftedProximableFunction, l::AbstractVector{R}, u::AbstractVector{R}) where {R <: Real}
```

Comment:
Should we call this `set_bounds!()` ? When the user only passes `∆`, we could have an alias:
```julia
set_radius!(ψ, Δ) = set_bounds!(ψ, -Δ, Δ)
```

Reply:
I modified the name of the function as it gives a better description of what really makes the function. However I didn't make an alias since I don't know how to handle the type issue it would raise.